.pre-commit-hooks.yaml

```diff
@@ -1,6 +1,6 @@
 -   id: python-check-blanket-noqa
     name: check blanket noqa
-    description: 'Enforce that `noqa` annotations always occur with specific codes'
+    description: 'Enforce that `noqa` annotations always occur with specific codes. Sample annotations: `# noqa: F401`, `# noqa: F401, W203`'
```

Comment:
oops, I only said a space after the colon, not after the period

Reply:
I'm a little lost here. Do you mean before `Sample`?

Reply:
I told you to change two things, and you changed three things for some reason.  there should not be a space after the comma